code/game/objects/items/devices/radio/headset.dm

```diff
@@ -66,6 +67,7 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 /obj/item/radio/headset/talk_into(mob/living/M, message, channel, list/spans, datum/language/language, list/message_mods)
 	if (!listening)
 		return ITALICS | REDUCE_RANGE
+	SEND_SIGNAL(src, COMSIG_RADIO_UPLINK_MESSAGE, M, message)
```

Comment:
Send this signal on the base radio type, and generalise the name to not be specific to uplinks

Reply:
oh!